media/js/mozorg/home/takeover-2016.js

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
```

Comment:
Perhaps... this was reused from last year's takeover so I didn't rethink it much. Though with the script loaded at the end of the body would we gain much by firing it before DOMready?